I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 10, in script
    define you = Characterr("You")
  File "game/script.rpy", line 10, in <module>
    define you = Characterr("You")
NameError: name 'Characterr' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 10, in script
    define you = Characterr("You")
  File "C:\Users\Lowie Suarez\Downloads\renpy-8.3.7-sdk\renpy\ast.py", line 2248, in execute
    self.set()
  File "C:\Users\Lowie Suarez\Downloads\renpy-8.3.7-sdk\renpy\ast.py", line 2262, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\Lowie Suarez\Downloads\renpy-8.3.7-sdk\renpy\python.py", line 1211, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 10, in <module>
    define you = Characterr("You")
NameError: name 'Characterr' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.7.25031702
DoctorsOrder 1.0
Mon Mar 24 19:29:17 2025
